base/src/main/java/butter/droid/base/data/DataBindModule.java

```diff
@@ -17,15 +17,16 @@
 
 package butter.droid.base.data;
 
-import butter.droid.provider.base.ProviderScope;
+import java.util.Set;
+
 import dagger.Module;
 import dagger.multibindings.Multibinds;
-import java.util.Set;
 import okhttp3.Interceptor;
 
 @Module
 public interface DataBindModule {
 
-    @Multibinds @ProviderScope Set<Interceptor> provideIntercptors();
+    @Multibinds
+    Set<Interceptor> provideIntercptors();
```

Comment:
`@PeroviderScope` should stay in as it is important for third parties implementing ther own providers.

Reply:
I removed as I updated Dagger and it does not allow Scopes on Multibinding anymore